archey/test/entries/test_archey_disk.py

```diff
@@ -128,6 +128,7 @@ def test_disk_df_output_dict(self, run_mock):
                 "/dev/nvme0n1p1                523248     35908        487340       7% /boot",
                 "/dev/sda1                       1624        42          1582       1% /what is  this",             # pylint: disable=line-too-long
                 "map auto_home                      0         0             0     100% /System/Volumes/Data/home",  # pylint: disable=line-too-long
+                "/Applications/Among Us.app/Wrapper 0         0             0     100% /System/Volumes/Data/game",  # pylint: disable=line-too-long
```

Comment:
@lourinaldi Could you replace this line by the one on your system so as to propose/verify behavior against a real-world example ? Thanks 🙇

Reply:
Up @lourinaldi !

Reply:
@HorlogeSkynet sorry, I just got back from vacation. Do you still need testing done?

Reply:
No problem @lourinaldi, just your advise on the test case that I added to check this behavior is properly handled.